server/src/workflow-management/record.ts

```
@@ -0,0 +1,231 @@
+        }
+
+        let currentPage = await browser.getCurrentPage();
+        if (!currentPage) {
+            throw new Error('Could not create a new page');
+        }
+
+        const workflow = AddGeneratedFlags(recording.recording);
+        const interpretationInfo = await browser.interpreter.InterpretRecording(
+            workflow,
+            currentPage,
+            (newPage: Page) => currentPage = newPage,
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)